dpnp/backend/extensions/blas/gemm.cpp

```diff
@@ -320,15 +310,9 @@ std::tuple<sycl::event, sycl::event, bool>
     const char *b_typeless_ptr = matrixB.get_data();
```

Comment:
Should we declare `is_row_major` with `constexpr` in case of `defined(USE_ONEMATH_CUBLAS)` above?